Remove yay dependency from Dockerfile

At present the installation of `yay` seems to break a docker build. Overall `yay` is not used in the Dockerfile, thus it seems to be safe to skip its installation.

```
Step 18/39 : RUN makepkg -si --noconfirm
 ---> Running in f806b3cd0f55
==> Making package: yay 10.0.0-1 [...]
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
[...]
==> Validating source files with sha256sums...
    yay-10.0.0.tar.gz ... Passed
==> Extracting sources...
  -> Extracting yay-10.0.0.tar.gz with bsdtar
==> Starting build()...
go build -v -mod=mod -ldflags '-s -w -linkmode external -extldflags "-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now" -X "main.yayVersion=10.0.0" -X "main.localePath=/usr/share/locale/"' -modcacherw -trimpath -o yay
[...]
github.com/Jguer/yay/v10/pkg/completion
github.com/Jguer/yay/v10
==> Entering fakeroot environment...
==> Starting package()...
msgfmt po/pt.po -o po/pt.mo
make: msgfmt: No such file or directory
make: *** [Makefile:150: po/pt.mo] Error 127
==> ERROR: A failure occurred in package().
    Aborting...
The command '/bin/sh -c makepkg -si --noconfirm' returned a non-zero code: 4
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -53,11 +53,6 @@ RUN chown arch:arch /home/arch
 
 USER arch
 WORKDIR /home/arch
-RUN git clone https://aur.archlinux.org/yay.git
-WORKDIR /home/arch/yay
-RUN makepkg -si --noconfirm
-
-WORKDIR /home/arch
 RUN git clone https://github.com/corpnewt/gibMacOS.git
 WORKDIR /home/arch/gibMacOS
 RUN perl -p -i -e 's/print("Succeeded:")/exit()/' ./gibMacOS.command

